source/vir/src/prune.rs

```diff
@@ -103,6 +103,14 @@ fn traverse_reachable(ctxt: &Ctxt, state: &mut State) {
     loop {
         let ft = |state: &mut State, t: &Typ| {
             match &**t {
+                TypX::StrSlice => {
```

Comment:
Please add a note that this is temporary, until we support adding specification for `std`.